packages/vue/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/vue",
-  "version": "2.1.6",
+  "version": "2.1.7-alpha.1",
```

```suggestion
  "version": "2.2.0-alpha.1",
```
To keep aligned with minor bumps from these changes